arreglo/src/main/java/com/globant/model/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.globant.model;
+
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue (strategy = GenerationType.AUTO)
+    private Integer id;
+    private String name;
+
+    public Order(Integer id, String name) {
```

Comment:
remove all geters and setters use the @DaTa from lombok and use the @NoArgsConstructor & @AllArgsConstructor aswell